### ISSUE TITLE
help about train SRWGANGP

### ISSUE THREAD

**Issue by DuongNo** [kolak33/SRWGANGP#3]
Dear kolak33

i have try train gan following guide of you on https://github.com/kolak33/SRWGANGP
but when i train model can not  converge, (model just save weight at epochs 0 while i was trained 130 epochs, loss function value of discriminator decreate from (-0,3 -> - 0,4) )

so can you help add more guide or up model pretrained of SRWGANGP to git

thank you very much

**Comment by kolak33** [comment]
Hi, did you change any parameters from config file or hyperparameters in the code? Especially pretraining the generator for first 3 epochs? Unfortunately I don't have the trained model anymore.

**Comment by DuongNo** [comment]
Hi @kolak33 
i dont change any thing parameters. 
after first 3 epochs pretraining generator and epochs 1 of final training generator make image same previous epochs and quality not improve
i check in code :

				if self.best_discriminator_loss < discriminator_loss_epoch:
					self.save_weights(epoch)
					self.best_discriminator_loss = discriminator_loss_epoch
					self.no_improvement_wasserstein_loss = 0
					print('saved weights, discriminator loss improved.')

and i see loss of discriminator never decreate after epochs 0 of final traning => model can not save after epoch 0 of final training
- quality of image from generator very low

**Comment by kolak33** [comment]
If training only the generator, and it's not improving, then it might be an issue with images from train dataset not matching. Low quality image and corresponding high quality images should match. Other than that I'm not sure.

**Comment by DuongNo** [comment]
@kolak33 
if training only the generator, just ok
but when training generator and discriminator it's not improving

**Comment by kolak33** [comment]
Then I don't know what's happening :(, sorry. But if you find out how to solve the issue I'd like to hear:).
